GenderPayGap.WebUI/Views/ReportOverview/ReportOverview.cshtml

```diff
@@ -1,257 +1,284 @@
+                        RightValue = !Model.SizeOfOrganisation.HasValue ? "Not Provided" : $"{Model.SizeOfOrganisation.GetAttribute<DisplayAttribute>().Name}"
+                    }
+                }
+            };
+            
+            var linkToGenderPayGapInformationSection = new ReportOverviewSectionViewModel
+            {
+                Title = "Link to your gender pay gap information",
+                EditLink = Url.Action("ReportLinkToWebsiteGet", "ReportLinkToWebsite", new 
+                {
+                    encryptedOrganisationId,
+                    reportingYear = Model.ReportingYear
+                }),
+                Rows = new List<ReportOverviewSectionRowViewModel>
+                {
+                    new ReportOverviewSectionRowViewModel
+                    {
+                        Title = @<a class="govuk-link" href="@(Model.LinkToOrganisationWebsite)">@(Model.LinkToOrganisationWebsite)</a>,
+                        RightValue = Model.LinkToOrganisationWebsite == null ? "Not Provided" : ""
+                    }
+                }
+            };
+            
+            var reportOverviewSections = new ReportOverviewSectionsViewModel
+            {
+                Title = $"{formattedReportingYears} Reporting year",
+                Sections = new List<ReportOverviewSectionViewModel>
+                {
+                    hourlyPayQuarterPercentageSection,
+                    hourlyPayMeanAndMedianSection,
+                    bonusPayPercentageSection,
+                    bonusPayMeanAndMedianGenderPayGapSection,
+                    personResponsibleSection,
+                    employeeHeadcountSection,
+                    linkToGenderPayGapInformationSection
+                }
+            };
+
+            await Html.RenderPartialAsync("../Components/ReportOverview/ReportOverviewSections", reportOverviewSections);
+        }
+        
+        <div class="govuk-inset-text">
+            <p>
+                <span>On submission your gender pay gap information will be published on the Gender pay gap service.</span>
+                <br/>
+                <br/>
+                <span>
+                    Submission of inaccurate data will mean your employer has not complied with the gender pay gap reporting regulations.
+                </span>
+            </p>
+        </div>
+
+        <div class="govuk-button-group">
+            @(Html.GovUkButton(new ButtonViewModel
+            {
+                Text = "Discard draft",
+                Classes = $"govuk-button govuk-button--secondary govuk-!-margin-bottom-6",
+                Href = Model.DraftReturnExists ? 
+                    Url.Action("ReportDiscardDraftGet", "ReportDiscardDraft", new { encryptedOrganisationId, reportingYear = Model.ReportingYear}) 
+                    : null,
+                Disabled = !Model.DraftReturnExists
+            }))
+
+
+            @Html.GovUkButton(new ButtonViewModel
```

Comment:
I had a look and we could either wrap it into a form or have a javascript function. I prefer the former, we should also add a aria-label to it for accessibility

Reply:
Tested this with NVDA and the form causes no issues so all good there